environment.yml

```diff
@@ -1,16 +1,27 @@
-name: geo_deep_env
+name: geo_ml_env
 channels:
+  - pytorch
+  - nvidia
   - conda-forge
 dependencies:
-  - python==3.11.5
+  - coverage>=6.3.1
+  - geopandas>=0.14.4
+  - hydra-core>=1.2.0
   - pip
-  - gdal
-  - pystac>=0.3.0
+  - pynvml>=11.0
+  - pystac>=1.10.1
   - pytest>=7.1
+  - python>=3.11
+  - pytorch>=2.3
```

Comment:
why do we need pytorch and pytorch-cuda at the same time? pytorch-cuda can work on CPU if needed. Won't having two different versions cause conflicts?

Reply:
It is the recommended way to install pytorch in a conda env.: https://pytorch.org/